resources/views/project/editcomment.blade.php

```diff
@@ -0,0 +1,31 @@
+@extends('layouts.main_layout')
+
+@section('head')
+    <title>Edit Comment</title>
+@stop
+
+@section('mainBody')
+    <div class="row">
+        <div class="col-md-6 col-md-offset-3">
+            <h4>Edit Comment</h4>
+
+            <form class="form-group" method="POST" action="/project/{{ str_replace(' ', '-', $comment->project->title) }}/{{ $comment->id }}">
```

Comment:
I've update the project model with a function for this, I'll fix this when it comes time to pr my stuff